src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,168 @@
+(ns status-im2.contexts.share.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.share.style :as style]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.qr-code.qr :as qr]
+            [clojure.string :as string]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
+            [status-im.ui.components.list-selection :as list-selection]
+            [status-im.ui.components.react :as react]))
+
+(def ^:const profile-tab-id 0)
+(def ^:const wallet-tab-id 1)
+
+(defn copy-text-and-show-toast
+  [{:keys [text-to-copy post-copy-message]}]
+  (react/copy-to-clipboard text-to-copy)
+  (js/setTimeout
+   #(rf/dispatch [:share/show-successfully-copied-toast post-copy-message])
+   250))
+
+(defn header
+  []
+  [rn/view
+   [quo/button
+    {:icon                true
+     :type                :blur-bg
+     :size                32
+     :accessibility-label :close-activity-center
+     :override-theme      :dark
+     :style               style/header-button
+     :on-press            #(rf/dispatch [:hide-popover])}
+    :i/close]
+   [quo/text
+    {:size   :heading-1
+     :weight :semi-bold
+     :style  style/header-heading}
+    (i18n/label :t/share)]])
+
+(defn profile-tab
+  [window-width]
+  (let [multiaccount              (rf/sub [:multiaccount])
+        emoji-hash                (string/join "" (get multiaccount :emoji-hash))
+        qr-size                   (- window-width 64)
+        public-pk                 (get multiaccount :public-key)
+        profile-qr-url            (str "https://join.status.im/u/" public-pk)
+        port                      (rf/sub [:mediaserver/port])
+        key-uid                   (get multiaccount :key-uid)
+        emoji-hash-max-width      (* window-width 0.76)
+        link-to-profile-max-width (* window-width 0.70)
+       ]
+    [:<>
+     [rn/view {:style style/qr-code-container}
+      [qr/user-profile-qr-code
+       {:window-width window-width
+        :key-uid      key-uid
+        :qr-size      qr-size
+        :public-key   public-pk
+        :port         port}]
+      [rn/view {:style (style/profile-address-container qr-size)}
+       [rn/view {:style style/profile-address-column}
+        [quo/text
+         {:size   :paragraph-2
+          :weight :medium
+          :style  style/profile-address-label}
+         (i18n/label :t/link-to-profile)]
+        [rn/touchable-highlight
+         {:active-opacity   1
+          :underlay-color   colors/neutral-80-opa-1-blur
+          :background-color :transparent
+          :on-press         #(copy-text-and-show-toast
```

Comment:
rf/dispatch should be used

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/fef2121e678eaf935e5bc7ff3dab07acd094d099